client/views/incidentReports/incidentForm.jade

```diff
@@ -3,23 +3,24 @@ template(name="incidentForm")
     label.control-label.featured.container-flex.no-break
       h5 Incident Details
     .add-incident--wrapper
-      if documentId
-        input(type="hidden" name="articleId" value=documentId)
-      unless confirmation
+      unless documentFromTextContent
```

Comment:
I think this will cause problems when editing incidents on documents from text since there will be no articleId input. It would be better to update the inputs and article selector to handle text based documents than create separate logic for them.